internal/pkg/types/keyinfo_test.go

```diff
@@ -31,3 +31,16 @@ func TestKeyInfoMarshal(t *testing.T) {
 	assert.Equal(t, ki.Type(), kiBack.Type())
 	assert.True(t, ki.Equals(kiBack))
 }
+
+func TestBLSPublicKey(t *testing.T) {
+	tf.UnitTest(t)
+	t.Skip("Dragons: BLS is broken")
+
+	testKey, err := crypto.GenerateKey()
```

Comment:
This is not how you generate a BLS key. Use `bls.PrivateKeyGenerate()` and crypto_test.go.